script.js

```diff
@@ -0,0 +1,90 @@
+const hamburger = document.getElementById('hamburger-btn');
+const mobMenu = document.getElementById('mobile-menu');
+
+function myFunction(x) {
+  x.classList.toggle('change');
+  if (mobMenu.style.visibility === 'visible') {
+    mobMenu.style.visibility = 'hidden';
+    mobMenu.style.height = '0';
+    mobMenu.style.overflowY = 'hidden';
+  } else {
+    mobMenu.style.visibility = 'visible';
+    mobMenu.style.height = '100vh';
+    mobMenu.style.overflowY = 'auto';
+  }
+}
+
+hamburger.addEventListener('click', myFunction);
+
+const speakers = [
+  {
+    name: 'ALH. Aminat Ajao',
+    profession: 'Islamic musician',
+    description: 'Won the award of Best Female Islamic Artiste of the Year',
+    img: 'images/Aminat Ajao Obirere.png',
+  },
+  {
+    name: 'Malama Khadija',
+    profession: 'Social activist',
+    description: 'Islamic studies teacher and founder of the Muslim Women Peace Forum.',
+    img: 'images/Malama Khadija Gambo Hawaja.png',
+  },
+  {
+    name: 'Sheikh Ahmad Abubakar Gumi',
+    profession: 'Islamic cleric',
+    description: 'Mufti and mufassir at the Kaduna central mosque Sultan Bello.',
+    img: 'images/Sheikh Ahmad Abubakar Gumi.png',
+  },
+  {
+    name: 'Sheikh Sajid Umar',
+    profession: ' Mufti and Judge,',
+    description: 'Director of Islamic Development since 2010 and  played an integral part in the start-up of the New Muslim Academy ',
+    img: 'images/sajid-umer.png',
+  },
+  {
+    name: 'Sheikh Muyideen Ajani Bello',
+    profession: 'Islamic scholar',
+    description: ' Chairman, Northern States’ Council Missioners of the Ansar Ud Deen Society of Nigeria',
+    img: 'images/Shelkh-Muyideen-Ajani-Bello.png',
+  },
+  {
+    name: 'Sheikh Ahmad Alkurdy',
+    profession: 'Seminary Instructor',
+    description: 'He serves the community in multiple capacities and has been a regular guest speaker and lecturer at numerous Islamic Centers, colleges and events',
+    img: 'images/ahmad alkurdy.png',
+  },
+];
+
+const referencePosition = document.getElementById('main-program');
+
+const createspeakerStructureHTML = () => {
+  referencePosition.insertAdjacentHTML('afterend',
```

Comment:
- [ ]  kindly consider making sure that the  dynamic content is attached to the static HTML by selecting the target parent element and adding the generated HTML with the DOM's `appendChild()` or `innerHTML()` to meet the below requirement
> The dynamic content is attached to the static HTML by selecting the target parent element and adding the generated HTML with the DOM's `appendChild()` or `innerHTML()`. For example:

```
  const speakerContainer = document.getElementById("element-id")
  ...
  speakerContainer.appendChild(dynamicElements)
  // OR
  speakerContainer.innerHTML = dynamicElements
  // If used inside a loop the innerHTML method should use `+=` instead of just `=`
```